From: Arnaud Rebillout <elboulangero@gmail.com>
Date: Wed, 09 Dec 2020 09:38:34 +0700
Subject: Skip some pkg/signal linux tests

Issue mentioned upstream at <https://github.com/moby/moby/pull/40353>,
there was a tentative fix at <https://github.com/moby/moby/pull/40496>,
but the issue is still present.

    === Failed
    === FAIL: pkg/signal TestCatchAll (0.00s)
      signal_linux_test.go:32: assertion failed: urgent I/O condition (string) != hangup (string)
      signal_linux_test.go:32: assertion failed: hangup (string) != child exited (string)
      signal_linux_test.go:32: assertion failed: child exited (string) != illegal instruction (string)
      signal_linux_test.go:32: assertion failed: illegal instruction (string) != floating point exception (string)
      signal_linux_test.go:32: assertion failed: floating point exception (string) != child exited (string)

Index: docker/engine/pkg/signal/signal_linux_test.go
===================================================================
--- docker.orig/engine/pkg/signal/signal_linux_test.go
+++ docker/engine/pkg/signal/signal_linux_test.go
@@ -14,6 +14,7 @@ import (
 )
 
 func TestCatchAll(t *testing.T) {
+	t.Skip("DM - Skip flaky test")
 	sigs := make(chan os.Signal, 1)
 	CatchAll(sigs)
 	defer StopCatch(sigs)
@@ -53,6 +54,7 @@ func TestCatchAllIgnoreSigUrg(t *testing
 }
 
 func TestStopCatch(t *testing.T) {
+	t.Skip("DM - Skip flaky test")
 	signal := SignalMap["HUP"]
 	channel := make(chan os.Signal, 1)
 	CatchAll(channel)
